src/components/header/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../../pages/_app';
+
+const Header = () => {
+  return (
+    <header className="header">
+      <div className="header-container">
+        <div className="header__left-content">
+          <div className="logo-wrapper">
+            <img src="../assets/images/logo.png" alt="Leapfrog" />
+          </div>
+          <div className="header__app-info">
+            CV Builder
+          </div>
+        </div>
+        <div className="header__right-content">
+          <div className="profile">
+            <div className="profile-detail">
+              <div className="profile__name">Ribby Frog</div>
+              <div className="profile__status">Employee</div>
+            </div>
+            <div className="dropdown">
+              <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can place this SVG on your icon.js file.

Reply:
🆗